framework/helpers/BaseHtml.php

```diff
@@ -1427,29 +1423,36 @@ public static function activeCheckbox($model, $attribute, $options = [])
 
     /**
      * Generates a boolean input
-     * This method is mainly called by [[activeCheckbox()]] and [[activeRadio()]].
-     * @param string $type the input type. This can be either `radio` or `checkbox`.
+     * This method is mainly called by [[activeCheckbox()]] and
+     * [[activeRadio()]].
+     * @param string $type the input type. This can be either `radio` or
+     * `checkbox`.
      * @param Model $model the model object
-     * @param string $attribute the attribute name or expression. See [[getAttributeName()]] for the format
-     * about attribute expression.
+     * @param string $attribute the attribute name or expression. See
+     * [[getAttributeName()]] for the format about attribute expression.
      * @param array $options the tag options in terms of name-value pairs.
```

Comment:
they cause no harm and i am just applying an 80 characters stop to make it easier to read a long with the code.